easybuild/easyconfigs/c/CHERAB/CHERAB-1.3.0-intel-2020b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'CHERAB'
+version = '1.3.0'
+
+homepage = 'https://cherab.github.io/documentation/index.html'
+
+description = """CHERAB is a python library for forward modelling diagnostics
+ based on spectroscopic plasma emission."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['2ce0ee3784863cc281c4f8ca402eb43d491f39d6269619161da09508e36eda45']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('matplotlib', '3.3.3'),
+    ('Raysect', '0.7.1')
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = False
+
+sanity_check_commands = [
+    'touch %(installdir)s/lib/python%(pyshortver)s/site-packages/%(namelower)s/__init__.py && '
+    'python -c "import cherab"'
+]
+
```

Comment:
what is the reason to create this `__init__.py`? The default sanity checks work fine on my side without it. Since it should not be necessary, I suggest to remove it
```suggestion
```

Reply:
@lexming, I've patched CHERAB to get around this hack which was necessary for it to work correctly with some of our other software.  I've opened an issue on the CHERAB github repo to try and get this fixed upstream.  (See https://github.com/cherab/core/issues/350).